node-util/www/html/restorer.php

```diff
@@ -2,6 +2,24 @@
 
 list($blank, $uuid, $blank) = split("/", $_SERVER["PATH_INFO"]);
 if (preg_match('/^([0-9a-fA-F]{24,32}|\w+-\w+-\d+)$/', $uuid)) {
+    // Get the list of user agents to ignore
+    $node_conf = file('/etc/openshift/node.conf');
+    $useragent_ignore_list = array();
+    foreach ($node_conf as $conf_line) {
+      if (preg_match('/UNIDLER_IGNORE_LIST/', $conf_line)) {
+        $useragent_ignore_value = array();
+        preg_match('/UNIDLER_IGNORE_LIST=[\'"]*([^"\']+)[\'"]*/', $conf_line, $useragent_ignore_value);
+        $useragent_ignore_list = explode(",", $useragent_ignore_value[1]);
+        break;
+      }
+    }
+    // If the user agent matches one in the list, move on.
+    foreach ($useragent_ignore_list as $ignore) {
+      if (preg_match("/" . $ignore . "/", $_SERVER["HTTP_USER_AGENT"])) {
+        header('HTTP/1.0 403 Forbidden');
```

Comment:
I thought about that too. Is there any reason to? I guess we could log that a request was ignored for app with <uuid> with user-agent string <user-agent string>. Would that be useful?